resources/measures/hvac_nighttime_oa_controls/tests/new_measure_test.rb

```diff
@@ -0,0 +1,124 @@
+# insert your copyright here
+
+require 'openstudio'
+require 'openstudio/measure/ShowRunnerOutput'
+require 'minitest/autorun'
+require_relative '../measure.rb'
+require 'fileutils'
+
+class NewMeasureTest < Minitest::Test
+  # def setup
+  # end
+
+  # def teardown
+  # end
+
+  def test_number_of_arguments_and_argument_names
+    # create an instance of the measure
+    measure = NewMeasure.new
+
+    # make an empty model
+    model = OpenStudio::Model::Model.new
+
+    # get arguments and test that they are what we are expecting
+    arguments = measure.arguments(model)
+    assert_equal(1, arguments.size)
+    assert_equal('space_name', arguments[0].name)
+  end
+
+  def test_bad_argument_values
+    # create an instance of the measure
+    measure = NewMeasure.new
+
+    # create runner with empty OSW
+    osw = OpenStudio::WorkflowJSON.new
+    runner = OpenStudio::Measure::OSRunner.new(osw)
+
+    # make an empty model
+    model = OpenStudio::Model::Model.new
+
+    # get arguments
+    arguments = measure.arguments(model)
+    argument_map = OpenStudio::Measure.convertOSArgumentVectorToMap(arguments)
+
+    # create hash of argument values
+    args_hash = {}
+    args_hash['space_name'] = ''
+
+    # populate argument with specified hash value if specified
+    arguments.each do |arg|
+      temp_arg_var = arg.clone
+      if args_hash.key?(arg.name)
+        assert(temp_arg_var.setValue(args_hash[arg.name]))
+      end
+      argument_map[arg.name] = temp_arg_var
+    end
+
+    # run the measure
+    measure.run(model, runner, argument_map)
+    result = runner.result
+
+    # show the output
+    show_output(result)
+
+    # assert that it ran correctly
+    assert_equal('Fail', result.value.valueName)
+  end
+
+  def test_good_argument_values
```

Comment:
This measure has insufficient measure tests, make sure to add some tests that check that check test model inputs and/or outputs are changing as-expected.

Reply:
I added more tests, after our discussion. Thanks!